### PR TITLE
Feature: Accept Shared File invitation [ch17906]

### PR DESCRIPTION
**Pull request by perfectmak**
**CHANGELOG**
- Implement grpc handler for `HandleFilesInvitation`. 
- Sends notification for invitation reply

### CODE REVIEW

File: core/space/services/services_sharing.go [hunk 1]
```diff
@@ -291,3 +291,61 @@ func (s *Space) ShareFilesViaPublicKey(ctx context.Context, paths []domain.FullP
...
+	if accept {
+		invitation, err = s.tc.AcceptSharedFilesInvitation(ctx, invitation)
+	} else {
+		invitation, err = s.tc.RejectSharedFilesInvitation(ctx, invitation)
+	}
+	if err != nil {
+		return err
+	}
+
+	// notify inviter,  it was accepted
```
Comment (dmerrill6) [inline core/space/services/services_sharing.go:325]: How do we know it wasn't rejected?

**Review by dmerrill6 — APPROVED** [review]
LGTM, left a question

File: core/space/services/services_sharing.go (hunk 1 above)
Comment (jsonsivar) [reply core/space/services/services_sharing.go:325]: Looks like the accept/reject helpers above inject the result into the `invitation` object so it gets passed into the message body naturally.

**Review by jsonsivar — APPROVED** [review]
LGTM

**Pull request merged**